### ISSUE TITLE
Plugin is broken in Fedora 26

### ISSUE THREAD

**Issue by PeterSprague** [mapircamera/QGIS#13]
Couldn't load plugin MAPIR_Processing due to an error when calling its classFactory() method 

# installed python dependencies that were causing first errors
$ pip install scipy opencv

# next error
AttributeError: 'module' object has no attribute 'STARTUPINFO' 
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/qgis/utils.py", line 333, in startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/usr/home/xxx/.qgis2/python/plugins/MAPIR_Processing/__init__.py", line 34, in classFactory
    from .MAPIR_Processing import MAPIR_Processing
  File "/usr/lib64/python2.7/site-packages/qgis/utils.py", line 607, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/usr/home/xxx/.qgis2/python/plugins/MAPIR_Processing/MAPIR_Processing.py", line 29, in 
    from MAPIR_Processing_dockwidget import MAPIR_ProcessingDockWidget
  File "/usr/lib64/python2.7/site-packages/qgis/utils.py", line 607, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/usr/home/xxx/.qgis2/python/plugins/MAPIR_Processing/MAPIR_Processing_dockwidget.py", line 84, in 
    si = subprocess.STARTUPINFO()
AttributeError: 'module' object has no attribute 'STARTUPINFO'

Python version: 2.7.13 (default, Jun 26 2017, 10:20:05) [GCC 7.1.1 20170622 (Red Hat 7.1.1-3)] 
QGIS version: 2.18.10 Las Palmas, exported

**Comment by mapircamera** [comment]
Hi there,

Currently we only support Windows 8 or newer. Sorry for any inconvenience. The error you are getting is due to a line that suppresses command window popups in the Windows OS.

**Comment by PeterSprague** [comment]
Might want to wrap that code with a test for Windows, then the rest of us Unix/Linux types doing science could use with MapIR cameras.  A Windows only python plug-in for QGIS?

**Comment by mapircamera** [comment]
Yes, we've only gotten it running and tested on windows so far, that's not to say with a few tweaks it won't work on Linux, we just haven't had time to go through all of the lines that are windows specific. Like, for example, the exiftool.exe file that ships with the package will not run on Linux, so the plugin will have an Exiftool dependency and will need to reference the local Exiftool installation in the copyExif function. This is also true of DCraw if you plan to process DNGs. The reason it's "Windows only" is simply because we programmed it on Windows, and haven't had time to go in and see what breaks on Linux.

**Comment by PeterSprague** [comment]
OK, fair enough.

I commented out those couple of lines and used "pip install" to provide the python packages.
